Cargo.toml

```diff
@@ -29,6 +29,9 @@ members = [
 ]
 exclude = ["mpc-stark", "mpc-bulletproof"]
 
+[profile.test]
+debug-assertions = false
```

Comment:
This is necessary to prevent `hyper` from validating our headers in the `test` profile. We need to attach raw signatures in unit tests